src/core/Points.tsx

```diff
@@ -60,12 +60,12 @@ export class PositionPoint extends THREE.Group {
 
     const localThreshold = threshold / ((this.scale.x + this.scale.y + this.scale.z) / 3)
     const localThresholdSq = localThreshold * localThreshold
-    const rayPointDistanceSq = _ray.distanceSqToPoint(_position)
+    const rayPointDistanceSq = _ray.distanceSqToPoint(this.position)
 
     if (rayPointDistanceSq < localThresholdSq) {
       const intersectPoint = new THREE.Vector3()
-      _ray.closestPointToPoint(_position, intersectPoint)
-      intersectPoint.applyMatrix4(this.matrixWorld)
+      _ray.closestPointToPoint(this.position, intersectPoint)
+      intersectPoint.applyMatrix4(parent.matrixWorld)
```

Comment:
this imo imo not sure. the function of the positionpoint class is to render a headless group with its own matrix which is then used to position the instance. the parents matrix would be the instance itself (THREE.Points) but that one is the same for all points.

Reply:
You are only talking about line 68 right?